src/Data/Profunctor.purs

```diff
@@ -33,5 +34,11 @@ rmap b2c = dimap id b2c
 arr :: forall a b p. (Category p, Profunctor p) => (a -> b) -> p a b
 arr f = rmap f id
 
+wrapIso :: forall t a p. (Newtype t a, Profunctor p) => p t t -> p a a
+wrapIso = dimap wrap unwrap
+
+unwrapIso :: forall t a p. (Newtype t a, Profunctor p) => p a a -> p t t
```

Comment:
I think both are potentially useful, right? Actually, are these the wrong way around?